### ISSUE TITLE
AAC(6')-29b might have a typo in v3.0.6

### ISSUE THREAD

**Issue by rpetit3** [arpcard/amr_curation#30]
I'm not sure if this is the expected behavior, but AAC(6')-29b appears to have a typo in v3.0.6 of the CARD. 

```
# v3.0.6
>gb|NG_048576.1 101-502 (+)|+|100-502|ARO:3002584|AAC(6')-29b [Pseudomonas aeruginosa]

# v3.0.5
>gb|EU118148|+|2006-2402|ARO:3002584|AAC(6')-29b [Pseudomonas aeruginosa]
```

The part that sticks out to me is 

gb|NG_048576.1 **101-502 (+)**|+|100-502|ARO:3002584|AAC(6')-29b [Pseudomonas aeruginosa]

It might be a typo, but thought you should check it out. 

This was discovered during usage with Ariba (related issue https://github.com/sanger-pathogens/ariba/issues/284)

Please let me know if there is anything I can do to help!

Edit:
I think this is what it should be:
```
>gb|NG_048576.1|+|100-502|ARO:3002584|AAC(6')-29b [Pseudomonas aeruginosa]
```

**Comment by tseemann** [comment]
The definition for that gene has moved from a Genbank record to a NG_ locus fromt he NCBI AMR Bioproject.

**Comment by tseemann** [comment]
CONFIRMED BUG:
```

--------------------- WARNING ---------------------
MSG: No whitespace allowed in FASTA ID [card~~~AAC(6')-29b~~~NG_048576.1 101-502 (+):100-502~~~aminoglycoside]
---------------------------------------------------
```

**Comment by arpcard** [comment]
Dang, we almost were bug free this round! Looking into this, it is a very odd error given our set up.

**Comment by tseemann** [comment]
Murphy's Law :)

**Comment by arpcard** [comment]
Issue resolved with today's release of CARD 3.0.7
